packages/server/package.json

```diff
@@ -37,6 +38,7 @@
     "ansi_up": "5.0.0",
     "ast-types": "0.13.3",
     "base64url": "^3.0.1",
+    "better-sqlite3": "8.2.0",
```

Comment:
typically we document dependency bumps. Should this have a changelog entry to mention we added this?

Reply:
Addressed here: [`74784cd` (#26168)](https://github.com/cypress-io/cypress/pull/26168/commits/74784cdc6f6e80d38811e3b2f8d23c29704b9541)